modules/340-monitoring-kubernetes/templates/node-exporter/daemonset.yaml

```diff
@@ -87,6 +87,8 @@ spec:
           mountPath: /var/run/node-exporter-textfile
         - name: dockersock
           mountPath: /var/run/docker.sock
+        - name: containerddir
```

Comment:
If the whole containerd directory is mounted, I think the following contained config mount is not required.